content/3_Silver/Graph_Traversal.mdx

```diff
@@ -1165,3 +1262,81 @@ else:
 ### Graph Two-Coloring Problems
 
 <Problems problems="bip" />
+
+<Quiz>
+	<Quiz.Question>
+		What's the main difference between DFS and BFS?
+
+		<Quiz.Answer>
+			DFS is recursive, while BFS is iterative.
```

Comment:
provide a link to a sample implementation of iterative dfs since its nowhere to be found since no one uses it

Reply:
do you have a link